src/snowflake/connector/query_context_cache.py

```diff
@@ -89,9 +80,7 @@ def __hash__(self) -> int:
         _hash += (_hash * 31) + self._read_timestamp
         _hash += (_hash * 31) + self._priority
         if self._context:
-            _hash += (_hash * 31) + int.from_bytes(md5(self._context).digest(), "big")
-        else:
-            _hash += _hash * 31
+            _hash += (_hash * 31) + int.from_bytes(md5(self._context.encode('utf-8')).digest(), "big")
```

Comment:
Dumb question: How is this hash change related to the SerDe change?

Reply:
To be consistent with the GS and JDBC changes, here `context` is `str` field, previous implementation is `bytearray`.